crypto/s2n_signature.h

```diff
@@ -0,0 +1,24 @@
+/* Copyright 2014 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+#pragma once
+
+#include "tls/s2n_tls_parameters.h"
+
+typedef enum {
```

Comment:
Following the convention in `s2n_hash.h` that uses a typedef for hash algorithm values. This also gives better type safety so that it should require an explicit cast if you mix up a type somewhere.